lib/index.js

```diff
@@ -1097,6 +1101,11 @@ vmadm.stop = function vmStop(opts, callback) {
         args.push('-F');
     }
 
+    if (opts.timeout) {
+        // all args are required to be strings
+        args.push('-t', opts.timeout.toString());
+    }
+
```

Comment:
Is there a reason that we can always just pass this argument? What if it's not supported in vmadm? In those cases should we instead tell consumers that we can't honor it?